src/components/GitOpsListPage.tsx

```diff
@@ -26,7 +26,6 @@ const GitOpsListPage: React.FC = () => {
   const [appGroups, setAppGroups] = React.useState(null);
   const [emptyStateMsg, setEmptyStateMsg] = React.useState(null);
   const [namespaces, nsLoaded, nsError] = useK8sWatchResource<any[]>(projectRes);
-  // const [namespaces, nsLoaded, nsError] = useK8sWatchResource<K8sResourceKind[]>(projectRes);
   const baseURL = '/api/gitops/pipelines';
```

Comment:
Note, I harded coded this.  We have to revisit this baseURL.  Check the static plugin